roseWong/assignments/mergeIntervals/lc56/lc56.py

```diff
@@ -2,6 +2,19 @@
 
 # Given a list of intervals, merge all the overlapping intervals to produce a list that has only mutually exclusive intervals.
 
+
+# Edge case: if the list has only 1 or 0 intervals, we just return the list.
+# The list may not be sorted, so first step is to sort the list, using the
+# first element of each interval. 
+# Initialize an array of merged intervals which is what the function will return.
+# Initialize a start pointer which will point to the first element of the first interval'
+# in the sorted list.
+# Initialize and end pointer which will point to the second/last element of the first
+# interval in the sorted list
+#
+
+
```

Comment:
Then what happens? Even if the approach may not pan out, feel free to write one out anyway!